### PR TITLE
rbp: Fix up settings after layout changes

### PR DESCRIPTION
**Pull request by popcornmix**
The 3D settings got lost in the refactor.
Also fix a couple of cosmetics.

### CODE REVIEW

**Comment by popcornmix** [comment]
ping @jjd-uk

File: system/settings/rbp.xml [hunk 1]
```diff
@@ -27,12 +27,17 @@
           </control>
           <control type="edit" format="integer" />
         </setting>
+        <setting id="videoplayer.supportmvc" type="boolean" label="38027" help="38028">
```
Comment (jjd-uk) [inline system/settings/rbp.xml]: I think this belongs with the other 3D settings under:

 group id="5" label="14232"

As a follow up I will then look at whether the label for that group needs to change as it's currently "Stereoscopic 3D".

**Comment by jjd-uk** [comment]
Oops sorry about that, not sure how that happened. 
Other than change the group "videoplayer.supportmvc" belongs to, it fits with what I've done so +1 to shove straight in.

**Comment by popcornmix** [comment]
Moved the section. No need for jenkins.

**Pull request merged**